sympy/solvers/ode/tests/test_systems.py

```diff
@@ -781,6 +893,60 @@ def test_linear_3eq_order1_type4_slow():
     dsolve(eq1)
 
 
+@slow
+@XFAIL
+def test_linear_neq_order1_type2_fail():
```

Comment:
How long does this test take to run? I'm trying it now and it takes a long time. Perhaps it's too slow for travis.

Reply:
Is this XFAIL because the answer is incorrect or because `checksysodesol` fails?

Reply:
> Is this XFAIL because the answer is incorrect or because `checksysodesol` fails?

I don't know that since the answer is way too big, I would have to check it manually. It would take a while to verify.

But, an important observation is that when the solution is very big from solve, `checksysodesol` usually gives a failure so there is a chance that there might be some problem with `checksysodesol`.

Reply:
> How long does this test take to run? I'm trying it now and it takes a long time. Perhaps it's too slow for travis.

It passed the travis tests though. I can still add too slow for travis.

Reply:
I guess it's not too slow then

Reply:
Can we split this test into two separate tests like:
```
def _make_linear_neq_order1_type2():
    ...
    return eq, sol

def test_linear_neq_order1_type2_dsolve():
    ...

@slow
@xfail
def test_linear_neq_order1_type2_checksysodesol_fail():
    ...
```
If you aren't sure that the solution is correct then there should be a comment explaining that.